tests/unit-fail/pta-field-2.ref

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
It's a pointer created by & applied to a scalar, which is treated as a pointer into an array of a single element according to the standard. That it also happens to be a field in a struct isn't relevant. Any pointer arithmetic resulting in a pointer outside the bounds of an array (+ 1 past the end) is undefined.